scripts/environment/bootstrap-macos-10.sh

```diff
@@ -3,12 +3,15 @@ set -e -o verbose
 
 brew update
 
-brew install ruby@2.7 coreutils cue-lang/tap/cue
+brew install ruby@2.7 coreutils cue-lang/tap/cue protobuf
 
 gem install bundler
 
 echo "export PATH=\"/usr/local/opt/ruby/bin:\$PATH\"" >> "$HOME/.bash_profile"
 
+# Force the proto-build crate to avoid building the vendored protoc.
+echo "export PROTO_NO_VENDOR=1" >> "$HOME/.bash_profile"
```

Comment:
I think you need a similar `echo PROTOC_NO_VENDOR=1 >> "${GITHUB_ENV}"` for this bootstrap script.